packages/fondue/src/components/Tooltip/Tooltip.tsx

```diff
@@ -56,6 +56,9 @@ const handleTimeout = (callback: () => void, delay: number, timeoutId: number |
 
 const isPointerEventTypeMouse = (event: PointerEvent): boolean => event.pointerType === 'mouse';
 
+/**
+ * @deprecated Please use updated tooltip component from `@frontify/fondue/components`.
```

Comment:
Could you add the migration guide like the other `@deprecated`?

Reply:
added